orttraining/orttraining/test/python/orttraining_test_onnx_ops_ortmodule.py

```diff
@@ -0,0 +1,154 @@
+"""
```

Comment:
Is this test being invoked from any file?

Reply:
It is a standalone test. I checks the correctness for SoftmaxGrad and GatherGrad. There was no python unit test checking explicitely GatherGrad. I added these to check explicity the gradient of gradient operator on CPU and GPU on very small networks.

Reply:
Do we not want to invoke it in the CI pipelines?

Reply:
I assumed it was automatically added. I added the file to the list of test file to run.